src/utils/axios.ts

```diff
@@ -1,25 +1,41 @@
 import axios, { AxiosRequestConfig } from 'axios';
 // config
-import { HOST_API } from 'config-global';
+import { BASE_URL, QOBRIX_PROXY_URL } from 'config-global';
 
 // ----------------------------------------------------------------------
+const axiosInstance = axios.create({ baseURL: BASE_URL });
 
-const axiosInstance = axios.create({ baseURL: HOST_API });
+axiosInstance.interceptors.response.use(
+  (res) => res,
+  (error) => Promise.reject((error.response && error.response.data) || 'Something went wrong')
+);
+
+// ----------------------------------------------------------------------
```

Comment:
same

Reply:
made as common structure of the template

Reply:
fix